testing/src/test/java/com/hazelcast/samples/testing/junit5/CustomerServiceComponentTest.java

```diff
@@ -0,0 +1,118 @@
+package com.hazelcast.samples.testing.junit5;
+
+import com.hazelcast.client.test.TestHazelcastFactory;
+import com.hazelcast.config.Config;
+import com.hazelcast.core.HazelcastInstance;
+import com.hazelcast.samples.testing.Customer;
+import com.hazelcast.samples.testing.CustomerService;
+import com.hazelcast.samples.testing.HzCustomerService;
+import com.hazelcast.samples.testing.SQLCustomerMapStore;
+import com.hazelcast.samples.testing.ServiceException;
+import com.hazelcast.map.MapStore;
+import com.hazelcast.sql.HazelcastSqlException;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+import static com.hazelcast.test.HazelcastTestSupport.randomName;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+/**
+ * <p>
+ * Why This is a Component Test:
+ * <ul>
+ *  <li>Involves the component under test (HzCustomerService) and its private dependencies (SQLCustomerMapStore via Hazelcast)</li>
+ *  <li>Uses a real external dependency (H2) and real Hazelcast instance (not mocked)</li>
+ *  <li>Tests distributed behaviour: eviction, load and reload, failure of downstream dependencies</li>
+ * </ul>
+ * </p>
+ */
```

Comment:
```suggestion
/**
 * Why This is a Component Test:
 * <ul>
 *  <li>Involves the component under test (HzCustomerService) and its private dependencies (SQLCustomerMapStore via Hazelcast)</li>
 *  <li>Uses a real external dependency (H2) and real Hazelcast instance (not mocked)</li>
 *  <li>Tests distributed behaviour: eviction, load and reload, failure of downstream dependencies</li>
 * </ul>
 */
```

Not sure why this needs to live in a paragraph?